packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/EthERC20Bridge.sol

```diff
@@ -190,8 +192,14 @@ contract EthERC20Bridge is L1Buddy {
         uint256 maxGas,
         uint256 gasPriceBid
     ) external payable onlyIfConnected returns (uint256) {
-        string memory name = SafeERC20Namer.tokenName(erc20);
-        string memory symbol = SafeERC20Namer.tokenSymbol(erc20);
+        string memory name = "";
+        string memory symbol = "";
+        try ERC20(erc20).name() returns (string memory _name) {
+            name = _name;
```

Comment:
This return type isn't always a string, it can be bytes32 such as for the MKR token https://etherscan.io/address/0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2#readContract

I think the cleanest approach is to staticcall the `name()` function and check the return data length. If its padded to 32 bytes we decode it to a string.